python/gym_jiminy/rllib/gym_jiminy/rllib/ppo.py

```diff
@@ -155,6 +155,11 @@ def _compute_mirrored_value(value: torch.Tensor,
     """Compute mirrored value from observation space based on provided
     mirroring transformation.
     """
+    # Make sure value and mirror_mat are on the same device.
+    # This is needed for multi-GPU training.
+    if value.device != mirror_mat.device:
+        mirror_mat = mirror_mat.to(value.device)
```

Comment:
no need to check the device. it is a no-op if unecessary.

Reply:
Fixed it.